tests/sles4sap/netweaver_filesystems.pm

```diff
@@ -24,6 +24,14 @@ sub run {
     my $sap_dir = "/usr/sap/$sid";
     my ($proto, $path) = $self->fix_path(get_required_var('NW'));
     my $nfs_client_service_name = is_sle('16+') ? 'nfs-client.target' : 'nfs.service';
+    my $test_name = get_required_var('TEST');
+    $test_name =~ s/_(?:node\d+|supportserver)\z//i;
+    my $flavor = get_required_var('FLAVOR');
+    my $version = get_required_var('VERSION');
+    my $subdir = join '_', $test_name, $flavor, $version;
```

Comment:
I don't think these 2 variables are used elsewhere, so perhaps:
```
my $subdir = join '_', $test_name, get_required_var('FLAVOR'), get_required_var('VERSION');
```
could be good enough.